README.md

```diff
@@ -1,11 +1,99 @@
 # Topiary
 
-![CI Status](https://github.com/tweag/topiary/actions/workflows/ci.yml/badge.svg)
+![CI Status][ci-badge]
 
 Topiary aims to be a uniform formatter for simple languages, as part of
-the [Tree-sitter](https://tree-sitter.github.io/tree-sitter/) ecosystem.
-It is named after the art of clipping or trimming trees into fantastic
-shapes.
+the [Tree-sitter] ecosystem. It is named after the art of clipping or
+trimming trees into fantastic shapes.
+
+Topiary is designed for formatter authors and formatter users. Authors
+can create a formatter for a language without having to write their own
+parser. Users benefit from uniform code style and, potentially, the
+convenience of using a single formatter tool across multiple languages
+over their codebases.
+
+## Motivation
+
+The style in which code is written has, historically, been mostly left
+to personal choice. Of course, this is subjective by definition and has
+led to many wasted hours reviewing formatting choices, rather than the
+code itself. Prescribed style guides were an early solution to this,
+spawning tools that lint a developer's formatting and ultimately leading
+to automatic formatters. The latter were popularised by
+[`gofmt`][gofmt], whose developers had [the insight][gofmt-slides] that
+"good enough" uniform formatting, imposed on a codebase, with no scope
+for subjective adjustment, largely resolves these problems.
```

Comment:
```suggestion
[`gofmt`][gofmt], whose developers had [the insight][gofmt-slides] that
"good enough" uniform formatting, imposed on a codebase largely resolves these problems.
```
I don't think that this bits adds relevant information.

Reply:
Removed in cd20b49a278a